src/templates/ControlTemplate.jsx

```diff
@@ -28,14 +32,74 @@ export default function ControlTemplate({
     implementation,
     status,
     version,
+    remarks,
   } = control;
   const { responsibility, components } = component;
   const subtitle = `System Control: ${label} ${controlTitle}`;
   const existingPrivateNarrative = components.private.description || "";
 
+  const naErrorModalRef = createRef();
+  const handleNaErrorOpen = () =>
+    naErrorModalRef.current?.toggleModal(undefined, true);
+  const selectNaModalRef = createRef();
+  const handleSelectNaOpen = () =>
+    selectNaModalRef.current?.toggleModal(undefined, true);
+  const deselectNaModalRef = createRef();
+  const handleDeselectNaOpen = () =>
+    deselectNaModalRef.current?.toggleModal(undefined, true);
+
+  function handleDeselectModalClicks(e) {
+    if (e) {
+      deselectNaModalRef.current?.toggleModal(undefined, false);
+      document
+        .querySelector(".usa-accordion__content")
+        .setAttribute("hidden", "hidden");
+      document
+        .querySelector(".usa-accordion__button")
+        .removeAttribute("aria-expanded");
+      document
+        .querySelector(".usa-accordion__content")
+        .setAttribute("aria-hidden", "hidden");
+    } else {
+      deselectNaModalRef.current?.toggleModal(undefined, false);
+      setNotApplicable(true);
+    }
+  }
+
+  function handleSelectModalClicks(e) {
+    if (e) {
+      selectNaModalRef.current?.toggleModal(undefined, false);
+      document.querySelector("#remarks").removeAttribute("hidden");
+      document
+        .querySelector("#remarks")
+        .setAttribute("aria-expanded", "expanded");
+      document.querySelector("#remarks").removeAttribute("aria-hidden");
+    } else {
+      selectNaModalRef.current?.toggleModal(undefined, false);
+      setNotApplicable(false);
+    }
+  }
+
+  function handleEmptyRemarksModalClicks(e) {
+    if (e) {
```

Comment:
Question (non-blocking): This guard seems a little strange to me.  I'm assuming that you're fighting with something in the TrussWorks component but why would this function be called if there wasn't an event to trigger it?